tom_hermes/hermes.py

```diff
@@ -135,8 +128,7 @@ def get_broker_context_data(self, alerts):
     def _request_messages(self, parameters):
         logger.debug(f'HermesBroker._request_messages()  parameters: {parameters}')
         response = requests.get(f'{HERMES_API_URL}/nonlocalizedevents/',
-                                params={**parameters},
-                                headers=settings.BROKERS['HERMES'])
```

Comment:
I removed the headers you were setting from the BROKERS dict HERMES field before... not sure what this was for but get endpoints in hermes don't need any headers?

Reply:
thanks. that was inherited (and overlooked) from tom_scimma